ddsp/training/docker/user_interaction.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+"""Script for gathering input, building the image and submitting job"""
+
+import os
+
+def get_input():
+  """Gathers input from user"""
+  # Determining path for data retrieval
+  input_dict = dict()
+  data_path = input('Insert a path for data retrieving. '
+                      'This must be a GCS Bucket:\n')
+  input_dict['data_path'] = data_path
+
+  # Determining path for storing model, snapshots and summaries
+  save_dir = input('\nInsert a path for saving the model, '
+                    'snapshots and summaries. This must be a GCS Bucket:\n')
+  input_dict['save_dir'] = save_dir
+
+  # Determining path for restoring the model
+  restore_dir = input('\nInsert a path from which checkpoints will '
+                      'be restored before training. Skip for using '
+                      'the same path as for saving:\n')
+  if restore_dir == '':
+    restore_dir = save_dir
+  input_dict['restore_dir'] = restore_dir
+
+  config_path = input('\nInsert the path to a configuration file '
+                      'or skip for the default one:\n')
+  if config_path == '':
+    config_path = './config_single_vm.yaml'
+  input_dict['config_path'] = config_path
+
+  image_uri = input('\nInsert an IMAGE URI. '
+                    'The template is: gcr.io/<GCP_PROJECT_ID>'
+                    '/<IMAGE_REPO_NAME>:<IMAGE_TAG> :\n')
+  input_dict['image_uri'] = image_uri
+
+  job_name = input('\nInsert a job name:\n')
+  input_dict['job_name'] = job_name
+
+  region = input('\nInsert the region you want to train '
+                  'your model in or skip for the default '
+                  'value (europe-west4): ')
+  if region == '':
+    region = 'europe-west4'
+  input_dict['region'] = region
+
+  batch_size = input('\nInsert batch size '
```

Comment:
Since all these prompts below are very similar, you could write a function for them to remove code duplication.